flake.nix

```diff
@@ -1,6 +1,6 @@
 {
   inputs = {
-    nixpkgs.url = "github:NixOS/nixpkgs";
+    nixpkgs.url = "github:NixOS/nixpkgs/e9b7f2f";
```

Comment:
nit: the commit is already pinned in `flake.lock`, so the url shouldn't mention the commit